.github/workflows/ci.yml

```
@@ -3,166 +1,218 @@
+      - uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          
+      - name: Check Python syntax
+        run: |
+          python -m py_compile src/**/*.py || true
+          find src -name "*.py" -exec python -m py_compile {} +
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)